acai-ios/MenuItemCustomizationViewController.swift

```diff
@@ -7,15 +7,90 @@
 //
 
 import UIKit
+import IGListKit
+
+protocol dismissMenuItemViewController: class {
+    func dismissMenuItemViewController()
+}
 
 class MenuItemCustomizationViewController: UIViewController {
 
+    // MARK: View vars
+    var collectionView: UICollectionView!
+    var collectionViewListAdapter: ListAdapter!
+    
+    var menuItem: MenuItem!
+    var customizationOptions: [Any]!
+    
     // MARK: View Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .red
+        view.backgroundColor = .white
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.backgroundColor = .white
+        collectionView.showsVerticalScrollIndicator = false
+        view.addSubview(collectionView)
+        collectionView.snp.makeConstraints { (make) in
+            make.top.equalToSuperview()
+            make.bottom.equalToSuperview()
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
+        }
+        
+        collectionViewListAdapter = ListAdapter(updater: ListAdapterUpdater(), viewController: self, workingRangeSize: 0)
+        collectionViewListAdapter.collectionView = collectionView
+        collectionViewListAdapter.dataSource = self
+    }
+}
 
-        // Do any additional setup after loading the view.
+extension MenuItemCustomizationViewController: ListAdapterDataSource {
+    func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        
+        menuItem = Acai.testBowl
+        
+        customizationOptions = menuItem.customizationOptions
+        var list = [] as [ListDiffable]
+        list.append(Acai.menuItemHeaderImageIdentifier as ListDiffable)
+        list.append(menuItem)
+        list.append(Acai.menuItemCustomizationOptionsIdentifier as ListDiffable)
+        list.append(Acai.menuItemButtonCollectionIdentifier as ListDiffable)
+        return list
     }
+    
+    func listAdapter(_ listAdapter: ListAdapter, sectionControllerFor object: Any) -> ListSectionController {
+        if let object = object as? String, object == Acai.menuItemHeaderImageIdentifier {
+            let headerImageSectionController = HeaderImageSectionController()
+            headerImageSectionController.delegate = self
+            headerImageSectionController.menuItem = menuItem
```

Comment:
does the header need the whole `menuItem` object or just a specific field? if its just a specific field, we should just pass that in instead